docs/user-guide/global-configurations/user-access.md

```diff
@@ -18,6 +18,7 @@ Devtron supports 5 levels of access:
 3. **Admin**: User with `admin` access can create, edit, delete and view permitted applications in permitted projects.
```

Comment:
**Admin**: User with `admin` privileges can create, edit, delete, and view permitted applications in permitted projects.